transport/rif_comms/layer.py

```diff
@@ -44,7 +46,27 @@ def construct_light_clients_nodes(self, config):
             sys.exit(1)
 
     def light_client_onboarding_data(self, address: Address) -> dict:
-        pass
+        return {
+            "transport_mode": self.transport_type,
+        }
 
     def register_light_client(self, raiden_api: 'RaidenAPI', registration_data: dict) -> TransportNode:
```

Comment:
something isn't right with the types hints here. the abstract transport layer has a matching signature for this method, but a `TransportNode` is not what's being returned.

one of these 2 is wrong

Reply:
this applies to the corresponding matrix method as well

Reply:
removed

Reply:
removed